source/includes/common/common-deploy.rst

```diff
@@ -115,7 +115,7 @@ Include any other environment variables as required for your local deployment.
       .. code-block:: shell
          :class: copyable
 
-         mc alias set http://localhost:9000 myminioadmin minio-secret-key-change-me
+         mc alias set myminio http://localhost:9000 myminioadmin minio-secret-key-change-me
 
       Replace ``myminioadmin`` and ``minio-secret-key-change-me`` with the :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD` values in the environment file specified to the container.
```

Comment:
We need to add a line here:
```suggestion
   - Replace ``myminio`` with the desired name to use for the alias.
   - Replace ``myminioadmin`` with the :envvar:`MINIO_ROOT_USER` value in the environment file specified to the container.
   - Replace ``minio-secret-key-change-me`` with the :envvar:`MINIO_ROOT_PASSWORD` value in the environment file specified to the container.
```

Also converting to a list to mimic how we do replacements in other code blocks.

Reply:
Addressed. Here is what the section looks like with my change:
![Screenshot from 2024-09-23 11-11-30](https://github.com/user-attachments/assets/f632db7c-d8e6-4071-9d60-0e6f157c4024)